force-app/main/default/permissionsets/Global_Growth_Base_Permission_Set.permissionset-meta.xml

```diff
@@ -444,16 +449,6 @@
         <field>Lead.LeadSource</field>
         <readable>true</readable>
     </fieldPermissions>
-    <fieldPermissions>
-        <editable>true</editable>
-        <field>Lead.Lead_Gen_Source_Training_Event__c</field>
-        <readable>true</readable>
-    </fieldPermissions>
-    <fieldPermissions>
-        <editable>true</editable>
-        <field>Lead.Master_Instructor_Lead_Generator__c</field>
-        <readable>true</readable>
-    </fieldPermissions>
     <fieldPermissions>
```

Comment:
@rthomp-prosci oops--we don't want to remove access from these fields

Reply:
These permissions are now restored. Thanks, @jnelson-prosci!